services-mail.md

```diff
@@ -204,6 +204,14 @@ Embedding inline images into your e-mails is typically cumbersome; however, ther
         <img src="{{ message.embed(pathToFile) }}">
     </body>
 
+If you are planning to use queued emails make sure that the path of the file is absolute. To achive that you can simply use the `app` [markup](https://octobercms.com/docs/markup/filter-app):
```

Comment:
Change `markup` to `filter` please, and link `app filter`, not just `filter`.
The URL needs to be relative from this files location `../markup/filter-app`
Typo on `achive`, should be `achieve`.